main.yml

```diff
@@ -12,9 +12,11 @@
 
   roles:
     - { role: installdocker, when: "{{ installdocker | default('False')}}" }
+    - { role: importdb, when: "{{ importdb | default('False')}}" }    
```

Comment:
Is it intended to do this in 1 line? This would mean the same thing without brackets:
``` yaml
- role: installdocker
  when: "{{ installdocker | default('False')}}"
```